webapp/src/client/app/modules/login/login.component.ts

```diff
@@ -77,7 +78,7 @@ export class LoginComponent extends BaseComponent implements OnInit {
   }
 
   private handleAutoLoginError(error): Observable<boolean> {
-    if (error.status && error.status !== '401') {
+    if (error?.status !== 401) {
```

Comment:
it's an integer instead of a string

rxjs catch error does not infer the type of the error